### PR TITLE
[Feature-1493][web,admin] remove flink jar settings to flink cluster manage

### LINKED ISSUE #1493: [Feature][Flink Settings] remove flink jar settings to flink cluster manage
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dlink/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

For the application mode under multi cluster, you need to adjust the jar file path multiple times（对于多集群下的application模式，需要多次调整jar文件路径）
image

### Use case

It is recommended to put it here. Each cluster corresponds to the relevant jar configuration file（推荐放入这里面，每一个集群对应相关的jar配置文件）
image

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zackyoungh**
## Purpose of the pull request
issue #i493 remove flink jar settings to flink cluster manage

image

### CODE REVIEW

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java [hunk 1]
```diff
@@ -257,13 +259,14 @@ private void loadDocker(
...
-
-        gatewayConfig.getAppConfig().setUserJarParas(params.split(" "));
+        //        String params =
+        //                SystemConfiguration.getInstances().getSqlSubmitJarParas()
+        //                        + buildParas(
+        //                                taskId,
+        //                                dockerConfig.getOrDefault("dinky.remote.addr",
+        // "").toString());
+        //
+        //        gatewayConfig.getAppConfig().setUserJarParas(params.split(" "));
```
Comment (leechor) [inline dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: should delete.

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java [hunk 2]
```diff
@@ -962,15 +965,14 @@ private JobConfig buildJobConfig(Task task) {
...
-                            "userJarMainAppClass",
-                            systemConfiguration.getSqlSubmitJarMainAppClass());
+                    Object sqlSubmitJarPath =
+                            Opt.ofBlankAble(gatewayConfig.get("userJarPath"))
+                                    .orElseThrow(
+                                            () ->
+                                                    new DinkyException(
+                                                            "application 模式支持需要在 注册中心->集群管理->集群配置管理 填写jar路径。"));
+                    gatewayConfig.put("userJarParas", buildParas(config.getTaskId()));
+                    gatewayConfig.put("userJarMainAppClass", "org.dinky.app.MainApp");
```
Comment (leechor) [inline dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java:970]: the user jar must contain this `MainMapp` class?

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java [hunk 3]
```diff
@@ -962,15 +965,14 @@ private JobConfig buildJobConfig(Task task) {
...
-                    gatewayConfig.put("userJarPath", systemConfiguration.getSqlSubmitJarPath());
-                    gatewayConfig.put(
-                            "userJarParas",
-                            systemConfiguration.getSqlSubmitJarParas()
-                                    + buildParas(config.getTaskId()));
-                    gatewayConfig.put(
-                            "userJarMainAppClass",
-                            systemConfiguration.getSqlSubmitJarMainAppClass());
+                    Object sqlSubmitJarPath =
+                            Opt.ofBlankAble(gatewayConfig.get("userJarPath"))
```
Comment (leechor) [inline dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: cast to `String` type better?

File: dinky-common/src/main/java/org/dinky/model/FlinkClusterConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * @author ZackYoung
+ * @since
+ */
+@Getter
+@Setter
+public class FlinkClusterConfiguration {
+    Type type;
+    String hadoopConfigPath;
```
Comment (leechor) [inline dinky-common/src/main/java/org/dinky/model/FlinkClusterConfiguration.java]: private better?

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java (hunk 2 above)
Comment (zackyoungh) [reply dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java:970]: Is not a user jar, it is dinky-app jar

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java (hunk 3 above)
Comment (zackyoungh) [reply dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: This behavior is just judgment

File: dinky-common/src/main/java/org/dinky/model/FlinkClusterConfiguration.java (hunk 4 above)
Comment (zackyoungh) [reply dinky-common/src/main/java/org/dinky/model/FlinkClusterConfiguration.java]: The default is private

Comment (leechor) [reply dinky-common/src/main/java/org/dinky/model/FlinkClusterConfiguration.java]: Default(Package – private)

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**